Kernel/Net/Ethernet/Ethernet.h

```diff
@@ -1,15 +1,17 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2020, the SerenityOS developers.
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
+#pragma GCC diagnostic ignored "-Warray-bounds"
```

Comment:
No reason to move that up, it only applies to this file and not to the includes of it